assignment6/rearrangingCars.py

```diff
@@ -0,0 +1,86 @@
+class Move:
+    """
+    Class that stores data about movement between two slots.
+    """
+    def __init__(self, origin, destination):
+        self.origin = origin
+        self.destination = destination
+
+    def __eq__(self, other):
+        return (self.origin == other.origin and
+                self.destination == other.destination)
+
+    def __str__(self):
+        return "Move the car from slot {} to slot {}.".format(self.origin,
+                                                            self.destination)
+
+
+def rearrangingCars(initalState, desiredState, toPrint = False):
```

Comment:
Very minor issue, the method naming convention in PEP8 suggests using all lowercase letter with underscores in between words [https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables](url)